bolides/bdf.py

```diff
@@ -13,16 +12,17 @@
 
 import matplotlib.pyplot as plt
 import matplotlib.dates as mdates
-from lightkurve import LightCurve, LightCurveCollection
 
-from . import API_ENDPOINT_EVENT, MPLSTYLE, ROOT_PATH
+from . import MPLSTYLE, ROOT_PATH
 from .utils import reconcile_input
 from .sources import glm_website, usg, pipeline, gmn, csv, remote
 
 _FIRST_COLS = ['datetime', 'longitude', 'latitude', 'source', 'detectedBy',
                'confidenceRating', 'confidence', 'lightcurveStructure',
-               'energy', 'energy_g16', 'energy_g17', 'brightness_g16', 'brightness_g17',
-               'brightness_cat_g16', 'brightness_cat_g17', 'impact-e', 'alt', 'vel']
+               'energy', 'energy_g16', 'energy_g17', 'energy_g18',
```

Comment:
It is not clear what is the different between energy and brightness. `bdf.describe()` doesn't help either. What is the difference? We should describe the difference to the users (and me!).

Reply:
I don't actually know! Energy and brightness are two different variables that appear in the JSON at https://neo-bolide.ndc.nasa.gov/service/event/public. Looking at the data, it looks like the "brightness" sometimes exactly matches the energy of the data point just before the peak (?) or the energy of a secondary peak (???). In the examples I've looked at, it doesn't match the peak energy. It's possible that the "brightness" value is something that, like the original energy value, should be removed. 

Is there someone who might know precisely what "brightness" means in the JSON export?

Reply:
Ah, this must be the peak brightness. When the web developers were setting this up, they were going to record the maximum group energy. I recommended they use a more robust number, like the median of the top 5 groups -- which effectively means the second or third brightest value. I suggest you change the name to `peak_brightness`. 

In hindsight, it might just have caused confusion to ask them to report a robust peak brightness.

Reply:
That makes sense. Would it be a good idea then to try to be consistent by renaming `energy_` to `integrated_energy_` and renaming `brightness_` to `peak_energy_` instead of `peak_brightness_`? That would make it more immediately clear what the variables are.

Reply:
Yes, that sounds better to me.